examples/SampleApp/android/build.gradle

```diff
@@ -42,6 +42,7 @@ allprojects {
             }
         }
         google()
+        jcenter()
```

Comment:
Isn't jcenter [discontinued](https://developer.android.com/studio/build/jcenter-migration)? Should we look into alternatives for this?